core/unit_test/incremental/Test12b_TeamScratch.hpp

```diff
@@ -116,6 +116,12 @@ struct TeamScratch {
 
 TEST(TEST_CATEGORY, IncrTest_12b_TeamScratch) {
   TeamScratch<TEST_EXECSPACE> test;
+#ifdef KOKKOS_ENABLE_OPENACC
+  // FIXME_OPENACC - scratch memory is not yet implemented in the OpenACC
+  // backend.
+  GTEST_SKIP() << "skipping since scratch memory is not yet implemented in the "
+                  "OpenACC backend";
+#endif
```

Comment:
```suggestion
#ifdef KOKKOS_ENABLE_OPENACC  // FIXME_OPENACC
  GTEST_SKIP() << "skipping since scratch memory is not yet implemented in the "
                  "OpenACC backend";
#endif
```